import_activity_events_retro.py

```diff
@@ -35,8 +35,16 @@ def env_or_default(variable_name, default_value):
     return default_value
 
 p = boto.provider.Provider("aws")
-AWS_ACCESS_KEY = env_or_default("AWS_ACCESS_KEY", p.get_access_key())
-AWS_SECRET_KEY = env_or_default("AWS_SECRET_KEY", p.get_secret_key())
+AWS_IAM_ROLE = env_or_default('AWS_IAM_ROLE', None)
```

Comment:
Btw (tangent, sorry), it wasn't clear to me whether it was worth the effort to still maintain these old `_retro` versions of the import scripts. I *think* they were written when we wanted to import data from an older format to the current schema, but it's a very vague memory and I might be wrong about that. They've certainly not been used in the last year.

Maybe we should just delete them, what do you think?

Reply:
(talking to myself now) `git log` gives points at 5b0f1dc639b0ee77fd160924bd55cfaa011f6da6, which leads to [this PR](https://github.com/mozilla/fxa-activity-metrics/pull/17). So I think we're good to delete these `_retro` import scripts, it's been over 2 years now and nobody is interested in the old data any more.